### PR TITLE
allow="fullscreen" rather than allow-full-screen

### PR DESCRIPTION
**Pull request by irvinebroque**
refs PCX-6713

Screenshot 2023-04-24 at 11 56 04 PM

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#allowfullscreen

### CODE REVIEW

**Comment by irvinebroque** [comment]
More instances of this here: https://github.com/search?q=repo%3Acloudflare%2Fcloudflare-docs%20allow-full-screen&type=code cc @dcpena

**Review by dcpena — APPROVED** [review]

**Review by third774 — APPROVED** [review]

**Comment by irvinebroque** [comment]
Doesn't fix the issue. See: https://7a18f480.cloudflare-docs-7ou.pages.dev/stream/get-started/

File: content/stream/get-started.md [hunk 1]
```diff
@@ -102,8 +102,7 @@ The embed code above can also be found in the [Cloudflare dashboard](https://das
       src="https://iframe.videodelivery.net/5d5bc37ffcf54c9b82e996823bffbb81?muted=true"
       title="Example Stream video"
       frame-border="0"
-      allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
-      allow-full-screen></iframe>
+      allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture fullscreen"></iframe>
```
Comment (third774) [inline content/stream/get-started.md]: Looks like there's a semi-colon missing after `picture-in-picture`, sorry for not spotting that sooner!

Comment (irvinebroque) [inline content/stream/get-started.md]: ty nice catch!

```suggestion
      allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture; fullscreen"></iframe>
```

**Review by dcpena — APPROVED** [review]

**Pull request merged**